evalml/automl/automl_search.py

```diff
@@ -857,16 +858,34 @@ def _handle_keyboard_interrupt(self):
             else:
                 leading_char = ""
 
-    def search(self, show_iteration_plot=True):
+    def search(self, show_iteration_plot=True, timing=None):
```

Comment:
I think we should move this to be an argument in `AutoMLSearch.__init__` instead of `AutoMLSearch.search`. Reason being, we have two ways for users to run search. This is one of them, but we're trying to move more over to running the top level `search` method instead of manually instantiating `AutoMLSearch` first. With the argument living here, users have no access to the argument. 

If we move the arg to `AutoMLSearch.__init__` and add it to the top level search methods as well, that will ensure users have full access to controlling this.